### PR TITLE
Add content to media and publications

### PR DESCRIPTION
**Pull request by toucham**
**Changes:**
- add static content to media and publications page
- changed how `<image />` element is set
- all information in media page is from `<meta /> of the website
- make both pages reponsive to mobile browser
- added `useScreenType` hook that returns the current screentype of the browser (`desktop` or `mobile`)

**Notes:**
- We could use a page scraper to automatically extract information for the `<ContentCard />` component which is gotten from `<meta />` elements in the media website; however, this requires a backend.

### CODE REVIEW

**Comment by IanSaucy** [comment]
One possible option for auto-scraping would be to use a serverless function. Out of scope for this week but possible after the launch on friday.

**Review by IanSaucy — CHANGES_REQUESTED** [review]
Could you make the image boxes a fixed size and just crop images that don't fit? This happened on the Publication page with the different sized images. I also think we might want to pull slightly different images than those used currently. Trying to include an image that's a bit more visual maybe as is in the wireframe.

Also up for discussion, should the sizing be done in `px` or something responsive like `vh, em, rem` etc

**Comment by toucham** [comment]
> Could you make the image boxes a fixed size and just crop images that don't fit? This happened on the Publication page with the different sized images. I also think we might want to pull slightly different images than those used currently. Trying to include an image that's a bit more visual maybe as is in the wireframe.
> 
> Also up for discussion, should the sizing be done in `px` or something responsive like `vh, em, rem` etc

If we're planning to make the website work on mobile, I think it is best to use either `vh, vw, rem, %`. Only use `px` in specific situation like setting `max-width`. However, I also highly suggest that we should prioritize `rem` with `media-query` over other units like `vh,vw,%` that scale according to parent elements since it's really hard to control these units to make it look good on every size of monitor.  

Furthermore, it is probably best to make most elements have [`display: flex`](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) so that the elements size will scale accordingly to the parent elements/monitor size.

**Review by IanSaucy — CHANGES_REQUESTED** [review]
Pending updates for responsiveness

File: src/styles/themes.less [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+/*when code is compiled, less => css; therefore, variable can't be changed during runtime,
+so instead, @font-base-size is changed to 1rem and is changed according to font-size in html element
+*/
+html {
+  font-size: 16px
+}
+
+@media screen and (max-width: 900px) {
+  html {
+    font-size: 10px
```
Comment (IanSaucy) [inline src/styles/themes.less:13]: Can you make this 12px instead, I think 10px is a little small.

**Review by IanSaucy — CHANGES_REQUESTED** [review]
Looks good, the dynamic sizing messed up the mobile menu bar text but it's not that big of a deal. Just file a bug into the backlog on Trello to fix later.

Otherwise just increasing mobile to 12px base. Looks super good, thank you!

**Review by IanSaucy — APPROVED** [review]
lgtm

**Pull request merged**